### ISSUE TITLE
Letters in paths are converted to upper case

### ISSUE THREAD

**Issue by Daniel-Abrecht** [dokan-dev/dokany#466]
# Environment
 * Windows version: Windows 7 Professional Service Pack 1
 * Processor architecture: x86_64
 * Dokany version: v1.0.2
 * Library type (Dokany/FUSE): FUSE
 
# Check List
  - [x] I checked my issue [doesn't exist yet](https://github.com/dokan-dev/dokany/issues?utf8=%E2%9C%93&q=is%3Aissue)
  - [x] I can always reproduce the issue with the provided description below.
  - [x] I have updated Dokany to the [latest version](https://github.com/dokan-dev/dokany/releases) and have reboot my computer after.
  - [x] I tested one of the last [snapshot](https://github.com/dokan-dev/dokany/wiki/Build#user-snapshot) from appveyor CI

# Description
I currently try to compile a program which mounts nfs shares using fuse with dokan-fuse, and for nfs exports from case-insensitive filesystems, everything works fine. However, when I try to mount a case-sensitive file system, I can't access any files on it except if their path is completely uppercase. It seams like the letters in file paths are converted to uppercase letters at some point, and since it's a case-sensitive file system, it doesn't find the files since technically, they aren't there. I haven't checked the paths I get for every fuse operation, but It at least happens with every path I get when the getattr function is called. Is this a bug, or do I have to pass some options to dokan_fuse, or is support for case-sensitive file system just not jet implemented in dokan_fuse? Also, where are those paths converted to uppercase, I just can't figure it out.

**Comment by Liryna** [comment]
Hi @Daniel-Abrecht ,

Do you mount using a reparse point ? `C:\MyFolder` 
If yes, this could be releated to https://github.com/dokan-dev/dokany/issues/293

**Comment by Daniel-Abrecht** [comment]
Yes, I do use a reparse point. So it seams to be a the same issue indeed. Initially, I thought it would also happen with mounted drive letters, but that turned out to be a different problem. By the way, is there a way to remove left over volumes which have never been cleanly unmounted?

**Comment by Liryna** [comment]
what do you mean by cleanly unmounted ? application that did the mount crashed and has left a unclean reparse point directory ?

**Comment by Daniel-Abrecht** [comment]
No, on my first try to port the fuse-nfs program, I mounted an nfs share to a drive letter, but I had used the wrong stat struct in the program, and when I tried to open the volume the program crashed and the explorer froze. I wasn't able to unmount the drive, I coudn't close the explorer or kill the explorer process, and the VM didn't want to reboot either. So I manually turned it off and on again, and I couldn't mount the same share at the same drive letter in a working way. But nevermind, another reboot solved that.
